### PR TITLE
Fixed issue. Tested locally end to end. However Still need to add the…

### PR DESCRIPTION
**Pull request by rahuljoshitaulia**
#372

### CODE REVIEW

File: src/main/java/com/github/tomakehurst/wiremock/http/HttpClientFactory.java [hunk 1]
```diff
@@ -66,14 +70,27 @@ public static HttpClient createClient(
...
+                })
+                .useTLS()
+                .build();
+            } else {
+              return SSLContexts.custom()
+                .loadTrustMaterial(null, new TrustSelfSignedStrategy())
+                .loadKeyMaterial(trustStore, trustStoreSettings.password().toCharArray())
+                .useTLS()
+                .build();
+            }
```
Comment (rahuljoshitaulia) [inline src/main/java/com/github/tomakehurst/wiremock/http/HttpClientFactory.java]: @tomakehurst  please recommend an appropriate test case for this and I will add it in.

**Comment by rahuljoshitaulia** [comment]
@tomakehurst  I apologize in advance if you dont want me to file against master branch. I would like to use a patched version for now until you decide to pull this into master or beta. What is the best way to go about doing this?

**Comment by tomakehurst** [comment]
Hi, apologies for the very long radio silence. I'm now attempting to to make amends for my inattention to the PR backlog.

As you say, master is frozen now. However, I'd really like to have this change in 2.x. Would you be able to  make this change against the 2.0-beta branch and open a new PR?

**Comment by avmohan** [comment]
@tomakehurst - Is this merged in 2.x? If not, I'm ok with taking this up.

**Comment by tomakehurst** [comment]
No, it never got re-built against 2.x, so if you'd like to have a go, that would be great.

**Pull request closed without merging**